Sources/SwifterSwift/SwiftStdlib/RangeReplaceableCollectionExtensions.swift

```diff
@@ -79,14 +79,14 @@ public extension RangeReplaceableCollection {
         guard let index = try firstIndex(where: predicate) else { return nil }
         return remove(at: index)
     }
-
+    
     /// SwifterSwift: Remove a random value from the collection.
     @discardableResult
     mutating func removeRandomElement() -> Element? {
         guard let randomIndex = indices.randomElement() else { return nil }
         return remove(at: randomIndex)
     }
-
+    
```

Comment:
So, since nothing happen with auto correct, can you please remove those additions manually?